web/src/main/java/com/epam/esm/controller/TagController.java

```diff
@@ -14,23 +17,18 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
 import javax.validation.Valid;
-import java.util.List;
 
 /**
  * The controller to provide CRD operations on {@link TagDTO}.
  */
 @RestController
-@RequestMapping(value = "/v1/tags")
+@RequestMapping(value = "/v2/tags")
```

Comment:
I suggest replacing v2 with v1 everywhere because you in a development state and haven't exposed your old endpoints outside.